packages/adbcflightsql

```diff
@@ -0,0 +1,6 @@
+{
+  "package": "adbcflightsql",
+  "url": "https://github.com/paws-r/paws",
```

Comment:
I believe the URL should be https://github.com/apache/arrow-adbc

Reply:
Corrected.